packages/orbit-components/src/SkipLink/README.md

```diff
@@ -27,10 +27,10 @@ After adding import into your project you can use it simply like:
 
 The table below contains all types of props available in the SkipLink component.
 
-| Name        | Type                | Description                            |
-| :---------- | :------------------ | :------------------------------------- |
-| buttonLabel | `string`            | Description for screen readers.        |
-| **links**   | [`links[]`](#links) | An array specifying links to point to. |
+| Name      | Type                | Description                            |
+| :-------- | :------------------ | :------------------------------------- |
+| label     | `string`            | Description for screen readers.        |
```

Comment:
Tried it with screen reader.
The screen reader output changed. With applying the `label` to `aria-label` of `nav`. It will be read as follows:
- content of the link
- `aria-label` of `nav`
- navigation

Therefore it would make sense to document it or change the description of `label` so it's aligned with actual behaviour.